autoscale-api/src/main/resources/messages/messages.properties

```diff
@@ -32,6 +32,8 @@ autoscale.validation.time.negative.adjustment=Schedule-Based Autoscaling Node Co
 autoscale.notification.history.update=Autoscaling History Updated
 autoscale.notification.config.update=Autoscaling Config Updated
 
+autoscale.stopstart.initial.upscale.adjustment=Adjusting cluster to max node count for hostgroup ''{0}'' based on the autoscale policy.
```

Comment:
Can we change this message a bit?
Because there might be a partial upscale as well and in that case Adjusting cluster to max node count might be a bit confusing.

Reply:
Or can we include the node count value being upscaled?

Reply:
>Can we change this message a bit?
Because there might be a partial upscale as well and in that case Adjusting cluster to max node count might be a bit confusing.

SInce this is sent from periscope we cannot determine the type of upscale that happens. I think any info regarding partial scales can be propagated from the actual flows that take place in cloudbreak

>Or can we include the node count value being upscaled?

Yeah I think including that should be good